Signal/src/ViewControllers/OWSTableViewController.m

```diff
@@ -149,7 +149,6 @@ - (UITableViewCell *)customCell
     if (_customCellBlock) {
         return _customCellBlock();
     }
-    OWSAssert(0);
```

Comment:
What does it mean when this returns nil? 

Can you update the signature to return (nullable UITableViewCell *)?